outermysite/polls/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
```

Comment:
this grouping of imports is not perfect. `requests` could go together with bs4 and django imports. `imdbScraper` can go in the end